.catch(function(response) {
  /*异常抛出*/
  
})